Makefile

```diff
@@ -123,7 +123,9 @@ clients/mockclients/cii_client.go: clients/cii_client.go
 checks/mockclients/vulnerabilities.go: clients/vulnerabilities.go
 	# Generating MockCIIClient
 	$(MOCKGEN) -source=clients/vulnerabilities.go -destination=clients/mockclients/vulnerabilities.go -package=mockrepo -copyright_file=clients/mockclients/license.txt
-
+checks/mockclients/packagemanager.go: cmd/packagemanager_client.go
```

Comment:
How would this work? Since the interface is not exposed shouldn't the mock also live in the same package as the interface?

Reply:
Sounds good. I can make that change and then merge it in. Thanks